pages/common/mytop.md

```diff
@@ -1,7 +1,7 @@
 # mytop
 
 > Display MySQL server performance info like `top`.
-> More information: <http://www.mysqlfanboy.com/mytop-3>.
+> More information: <http://jeremy.zawodny.com/mysql/mytop/mytop.html>.
```

Comment:
Though ugly-looking, this link appears to be a good one.  From the page:

>mytop was developed and is maintained by Jeremy D. Zawodny (Jeremy@<!---->Zawodny.com)

Reply:
Do you mean <http://www.mysqlfanboy.com/mytop-3>? It neither works for me on linux, nor on android.

Reply:
No, I'm referring to the new one.  I was adding a note for future reference, in case someone thought that was some random blog post (because it looks like one).

Reply:
Odd, I guess so. Hopefully it won't go down again 😬

Reply:
I think we should keep the second one in this case, since it also seems to be more official.

Reply:
Looks like the existing link works again.

Reply:
@sbrl as I exlained above, I think we should still keep the new one since it seems to be more official.